cfgen/cfgen

```diff
@@ -1728,7 +1732,29 @@ def build_cluster(cluster_desc: Dict[str, Any]) -> Cluster:
                 proc_id = ConfProcess.build(conf, node_id, node, proc_t)
                 cluster.m0_clients[proc_id] = new_oid(ObjT.service)
 
+    aux_pools = {}
     for pool in cluster_desc['pools']:
+        parity_group_size = pool['data_units'] + 2 * pool['parity_units']
```

Comment:
This needs to change to pool['data_units'] + pool['parity_units'] + pool['spare_units'].

Reply:
> @suvratjoshi, will you be taking this PR forward?

Yes @mssawant once I get back on this.
Most probably I will be creating new PR which will have all these additions/changes.

Reply:
> This needs to change to pool['data_units'] + pool['parity_units'] + pool['spare_units'].

Yes. I had this change in my mind for this PR, as now N+K+S formula will be active. But thanks for keeping reminder here.
This change will be possible once PR #1545 is merged.